modules/ting/src/Search/TingSearchFieldFilter.php

```diff
@@ -33,7 +35,9 @@ class TingSearchFieldFilter implements BooleanStatementInterface {
   /**
    * Field value.
    *
-   * @var mixed
+   * If NULL the field is a boolean field.
```

Comment:
Does this hold? The constructor seems to set it to `self::BOOLEAN_FIELD_VALUE` which is not `NULL`.

Reply:
argh, leftover from when i started out with it being null.